FreeRTOS/FreeRTOS
# Improvement to UARTCommandConsole.c

Description
-----------
Removed the function scope **xPort** variable within FreeRTOS+CLI task in UARTCommandConsole.c in order for the task to use the static file scope **xPort** variable. Also, added a prototype of vOutputString.

Test Steps
-----------
Tested by running the FreeRTOS+CLI demo on MSP432

Checklist:
----------
- [] I have tested my changes. No regression in existing tests.
- [] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------
Fixes #548 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
